pio/ws2812/ws2812.c

```diff
@@ -123,7 +144,11 @@ int main() {
     bool success = pio_claim_free_sm_and_add_program_for_gpio_range(&ws2812_program, &pio, &sm, &offset, WS2812_PIN, 1, true);
     hard_assert(success);
 
+#if IS_800KHz
     ws2812_program_init(pio, sm, offset, WS2812_PIN, 800000, IS_RGBW);
```

Comment:
Perhaps it makes more sense to add e.g. `#define WS2812_FREQUENCY 800000` and then pass that to `ws2812_program_init`, rather than adding a `#if IS_800KHz` section?